gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.java

```diff
@@ -64,6 +64,10 @@ public abstract class LaunchConfigTask extends DefaultTask {
     // Disable C2 compilation for problematic structure in JDK 11.0.16, see https://bugs.openjdk.org/browse/JDK-8291665
     private static final ImmutableList<String> jdk11DisableC2Compile =
             ImmutableList.of("-XX:CompileCommand=exclude,sun/security/ssl/SSLEngineInputRecord.decodeInputRecord");
+    // UseContainerCpuShares was added in a patch release, thus IgnoreUnrecognizedVMOptions is required to avoid
+    // breaking distributions running with older JDKs.
+    private static final ImmutableList<String> forceUseContainerCpuShares =
+            ImmutableList.of("-XX:+IgnoreUnrecognizedVMOptions", "-XX:+UseContainerCpuShares");
```

Comment:
Updated with `IgnoreUnrecognizedVMOptions` because `UseContainerCpuShares` was added in the patch that changed behavior. It pains me a bit, but it's likely our best option.

oss latest-jdks is a bit behind, and didn't have the flag. Resolved that piece by merging https://github.com/palantir/gradle-jdks-latest/pull/59 but we need to be careful about implicit dependencies on specific jdk builds.